src/NServiceBus.Core.Tests/Msmq/MsmqQueueCreatorTests.cs

```diff
@@ -101,6 +101,33 @@ public void Should_make_queues_non_transactional_if_requested()
             Assert.AreEqual(isTransactional, queue.Transactional);
         }
 
+        [Test]
+        public void Msmq_will_add_everyone_and_anonymous_by_default()
+        {
+            var testQueueName = "MsmqQueueCreatorTests.MsmqDefaultPermissions";
+
+            DeleteQueueIfPresent(testQueueName);
+
+            var path = MsmqAddress.Parse(testQueueName).PathWithoutPrefix;
+
+            using (var queue = MessageQueue.Create(path))
+            {
+                MessageQueueAccessRights? everyoneAccessRights;
+
+                Assert.True(queue.TryGetPermissions(LocalEveryoneGroupName, out everyoneAccessRights));
+                Assert.True(everyoneAccessRights.HasValue, "User should have access rights");
+                Assert.AreEqual(MessageQueueAccessRights.GenericWrite, everyoneAccessRights, "Msmq should give 'everyone' write access by default");
```

Comment:
@ramonsmits note that MSMQ gives generic write and not "write" to everyone by default. I wonder if that is an issue?

But we decided to keep the code that sets it to write as well so that won't be an issue I guess